pybv/io.py

```diff
@@ -133,14 +129,27 @@ def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
 
     _chk_fmt(fmt)
 
-    if unit == 'μV':
-        # this is greek mu: μ
-        # https://www.compart.com/de/unicode/U+03BC
-        warnings.warn(
-            f"Encountered small greek letter mu: 'μ' in unit: {unit} ... "
-            f"converting to micro sign: 'µ': {unit.replace('μ', 'µ')}"
-        )
-        unit = 'µV'
+    # check unit is single str or list
+    if isinstance(unit, list):
```

Comment:
I wouldn't do this. Python uses duck typing, so I one should be able to expect that passing a tuple or numpy array should work as well.

Instead, I would _first_ check for `str` identity, and only if it's not a string, cast to a list via `list(unit)`, and then proceed.

Reply:
Updated in https://github.com/bids-standard/pybv/pull/66/commits/0d920ea736c64cb50e2917eb0b7dcb5a8df40624